README.md

```diff
@@ -1,86 +1,56 @@
-[Cozy](https://cozy.io) Drive for Desktop <sup>(beta)</sup>
-===========================================================
+# [Cozy](https://cozy.io) Drive for Desktop <sup>(beta)</sup>
+
+[![Cozy Logo](https://raw.github.com/cozy/cozy-setup/gh-pages/assets/images/happycloud.png)](https://cozy.io)
 
 [![Latest release](https://img.shields.io/github/release/cozy-labs/cozy-desktop/all.svg)](https://github.com/cozy-labs/cozy-desktop/releases)
 [![Licence](https://img.shields.io/github/license/cozy/cozy-drive.svg)](https://github.com/cozy/cozy-drive/blob/master/LICENSE)
 <br>
 [![Windows build](https://img.shields.io/appveyor/ci/cozy/cozy-desktop/master.svg?label=windows)](https://ci.appveyor.com/project/cozy/cozy-desktop/branch/master)
-[![Linux & macOS build]( 	https://img.shields.io/travis/cozy-labs/cozy-desktop/master.svg?label=mac/linux)](https://travis-ci.org/cozy-labs/cozy-desktop/branches)
+[![Linux & macOS build](https://img.shields.io/travis/cozy-labs/cozy-desktop/master.svg?label=mac/linux)](https://travis-ci.org/cozy-labs/cozy-desktop/branches)
 [![Code climate](https://img.shields.io/codeclimate/github/cozy-labs/cozy-desktop.svg)](https://codeclimate.com/github/cozy-labs/cozy-desktop)
 [![Coverage](https://img.shields.io/codecov/c/github/cozy-labs/cozy-desktop/master.svg?label=coverage)](https://codecov.io/gh/cozy-labs/cozy-desktop/list/master)
 
-## What is Cozy?
+Cozy Drive for Desktop allows you to synchronize the files stored in [your Cozy](https://cozy.io) with your laptop and/or desktop computer.
+It replicates your files on your hard drive and applies the changes you make to them on other synchronized devices and on [your online Cozy](https://github.com/cozy/cozy-stack).
+
+You can download it for Windows, macOS, and Linux on [**Cozy Drive for Desktop download page**](https://cozy.io/en/download/).
 
-![Cozy Logo](https://raw.github.com/cozy/cozy-setup/gh-pages/assets/images/happycloud.png)
+The [`cozy-desktop`](https://github.com/cozy-labs/cozy-desktop) repository is where we do development and there are many ways you can participate in the project, for example:
 
-[Cozy](https://cozy.io/) is a platform that brings all your web services in the same private
-space.  With it, your web apps and your devices can share data easily,
-providing you with a new experience. You can install Cozy on your own hardware
-where no one profiles you.
+-   [Submit bugs and feature requests](https://github.com/cozy-labs/cozy-desktop/issues) and help us verify as they are checked in.
+-   Review [source code changes](https://github.com/cozy-labs/cozy-desktop/pulls).
+-   Review the [documentation](https://github.com/cozy-labs/cozy-desktop/tree/master/doc) and make pull requests for anything from typos to new content.
 
+## Contributing
```

Comment:
This paragraph will be changed with the `doc/` documentations enhancement with a list of links pointing to the good ressource.